documentation/modules/ROOT/pages/configuration/event-flattening.adoc

```diff
@@ -10,18 +15,56 @@ toc::[]
 
 [NOTE]
 ====
-This SMT is supported only for the SQL database connectors, it does not work with the MongoDB connector.
-See xref:configuration/mongodb-event-flattening.adoc[here] for the MongoDB equivalent to this SMT.
+This single message transformation (SMT) is supported for only the SQL database connectors. For the MongoDB connector, see the xref:configuration/mongodb-event-flattening.adoc[documentation for the MongoDB equivalent to this SMT].
 ====
+endif::community[]
+
+ifdef::product[]
+[id="extracting-source-record-after-state-from-debezium-change-events"]
+= Extracting source record `after` state from Debezium change events
+endif::product[]
+
+A Debezium database change event has a complex structure that provides a wealth of information. Kafka records that convey Debezium change events contain all of this information. 
```

Comment:
Sure. I'll make sure the doc uses "data change event".

As for replacing instances of `Debezium` with `{productname}` -- In the upstream doc, I have searched for but am not finding an AsciiDoc attribute specification for the product name. Many doc source files use {prodname} and the published doc does indeed render "Debezium" but I cannot see where this is coming from. Can someone point me to the declaration? 

Also, downstream, we set `:prodname:` to `Integration`. I can override that for the downstream Debezium books and set `:prodname:` to `Debezium` for just those two books, or we can use an attribute with a different name upstream and then have no collision downstream. Downstream, there is already a definition of: 

`:ProductName: Debezium`

Do you want to use that upstream?

Reply:
I believe what Gunnar was referring to was using `{prodname}`.  

We have a few attributes which we define as _global_ and relatively _static_ in that they'd be used across all modules of a given playbook.  Our antora playbook files are maintained in a different repository, see _debezium.github.io_.

In that repository we define the following attributes:
```
# Global asciidoc attributes here, used across all versions of documentation
asciidoc:
  attributes:
    prodname: 'Debezium'
    context: 'debezium'
    jira-url: 'https://issues.redhat.com'
    # because of how handlebars templates work with page.attributes, this must be prefixed with "page-"
    page-copyright-year: '2020'
```

Reply:
Good to know! Thanks. We'll continue to use prodname downstream then